docs/source/changelog.md

```diff
@@ -7,6 +7,15 @@ list and release milestones.
 
 Release notes for the `space_packet_parser` library
 
+### v6.2.0 (Unreleased)
+
+- **BREAKING**: Migrated from Poetry to uv/hatchling for Python project management
```

Comment:
I don't think this is a breaking change? It is under the hood and should be transparent to any users. Specifically, when installing a wheel, this doesn't even have any impact at all, only while building it yourself.